vali_objects/utils/plagiarism_definitions.py

```diff
@@ -0,0 +1,219 @@
+
+from vali_objects.utils.plagiarism_events import PlagiarismEvents
+from sklearn.metrics.pairwise import cosine_similarity
+import heapq
+import numpy as np
+from vali_objects.vali_config import ValiConfig
+
+
+class FollowPercentage(PlagiarismEvents):
```

Comment:
nice, makes a lot of sense for each child plagiarism type to inherit from the parent